compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/UIKitTextInputService.uikit.kt

```diff
@@ -298,6 +301,32 @@ internal class UIKitTextInputService(
             sendEditCommand(FinishComposingTextCommand())
         }
 
+        override fun positionFromPosition(position: Long, offset: Long): Long {
+            val text = getState()?.text ?: return 0
+
+            if (position + offset >= text.lastIndex + 1) {
+                return (text.lastIndex + 1).toLong()
+            }
+            if (position + offset <= 0) {
+                return 0
+            }
+            var currentOffset = offset
+            var resultPosition = position.toInt()
+
+            val it = BreakIterator.makeCharacterInstance()
+            it.setText(text)
+
+            while (currentOffset > 0) { // Move forward
+                resultPosition = it.following(resultPosition)
+                currentOffset--
+            }
+            while (currentOffset < 0) { // Move backward
+                resultPosition = it.preceding(resultPosition)
+                currentOffset++
```

Comment:
Adding one to offset in unicode text is incorrect. Please use `CodePoint` extensions that I've added

Reply:
Ah. Does `offset` here mean just count of such operations. Then, let's write comment to parameters

Reply:
So, we don't need to use `CodePoint` extension. But shoud add a comment.
Correct ?

Reply:
if it's number of unicode chars and not byte-offset - yes.

Reply:
Added doc comment:
```
* @param offset a character offset from position. It can be a positive or negative value.
* Offset should be considered as a number of Unicode characters. One Unicode character can contain several bytes.
```